src/v0/sources/shopify/util.js

```diff
@@ -126,6 +126,7 @@ const getAnonymousIdFromDb = async (message, metricMetadata) => {
     try {
       anonymousId = await RedisDB.getVal(`${cartToken}`, 'anonymousId');
     } catch (e) {
+      logger.debug(`{{SHOPIFY::}} anonymousId get call Failed due redis error ${e}`);
```

Comment:
`logger.debug(`{{SHOPIFY::}} Failed to retrieve anonymousId from Redis due to an error: ${e}`);`